### PR TITLE
chore: improve documentation for the monitoring add-on

### PR DESCRIPTION
**Pull request by snusifer**
Changelog: none

Signed-off-by: Mikael Torp-Holte <mikael.torp-holte@northern.tech>

### CODE REVIEW

File: 01.Get-started/04.Deploy-a-container-update/docs.md [hunk 1]
```diff
@@ -88,7 +91,6 @@ On MacOS
 wget https://downloads.mender.io/mender-artifact/master/darwin/mender-artifact -O ${HOME}/bin/mender-artifact
 ```
 
-
```
Comment (oleorhagen) [inline 01.Get-started/04.Deploy-a-container-update/docs.md:91]: Nit: random whitespace removal not necessary

File: 01.Get-started/05.Monitor/docs.md [hunk 2]
```diff
@@ -162,32 +167,28 @@ events and notifies you in the event that a container was restarted.
...
+sudo docker run --name demo --rm -d alpine sleep infinity
 ```
 
-Create the file `dockerevents_testcontainer.sh` with the contents:
+Create the file `/etc/mender-monitor/monitor.d/available/dockerevents_testcontainer.sh` running the following command:
 
 ```bash
-DOCKER_CONTAINER_NAME=$CONTAINER_NAME
+sudo tee /etc/mender-monitor/monitor.d/available/dockerevents_testcontainer.sh <<EOF
+DOCKER_CONTAINER_NAME=demo
```
Comment (oleorhagen) [inline 01.Get-started/05.Monitor/docs.md:177]: Q:
Did we not conclude that this needs to be `DOCKEREVENTS_CONTAINER_NAME`?

**Review by oleorhagen — APPROVED** [review]
Seriously fine job here!!!

I'm impressed.

Medal awarded for distinguished service to the Mender project

:1st_place_medal:

File: 01.Get-started/05.Monitor/docs.md (hunk 2 above)
Comment (snusifer) [reply 01.Get-started/05.Monitor/docs.md:177]: Woops! That was the whole point of making a new PR haha

**Review by lluiscampos — COMMENTED** [review]
Thank you @snusifer for the nice improvements :top:

File: 01.Get-started/04.Deploy-a-container-update/docs.md [hunk 3]
```diff
@@ -160,20 +163,24 @@ it to your device.
...
+```bash
+sudo docker ps -a | head
+```
+> ```console
+> CONTAINER ID   IMAGE         COMMAND    CREATED          STATUS                      PORTS     NAMES
+> 72e6cf80fa6a   hello-world   "/hello"   45 seconds ago   Exited (0) 42 seconds ago             optimistic_shaw
+> ```
 
 The [Docker Update Module](https://hub.mender.io/t/docker/324?target=_blank) will
 download and run the specified images from e.g https://hub.docker.io.
```
[the remark this answers is not in the document]
Comment (oleorhagen) [reply 01.Get-started/04.Deploy-a-container-update/docs.md:186]: Just made it into a proper link.

File: 01.Get-started/04.Deploy-a-container-update/docs.md [hunk 4]
```diff
@@ -55,11 +55,12 @@ Once the installation script has finished, verify that you can run the
...
->```bash
->pi@raspberrypi:~$ sudo docker images
->REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
->```
+```bash
+sudo docker images
+```
+> ```console
+> REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+> ```
```
[the remark this answers is not in the document]
Comment (oleorhagen) [reply 01.Get-started/04.Deploy-a-container-update/docs.md:63]: resolved f2f, will leave as is. It looks good on a local docs site bringup.

**Comment by oleorhagen** [comment]
Edits by maintainers not allowed, had to open a new PR here:

https://github.com/mendersoftware/mender-docs/pull/1718

**Pull request closed without merging**